src/pytorch_lightning/strategies/sharded_spawn.py

```diff
@@ -19,10 +19,10 @@
 from torch.optim import Optimizer
 
 import pytorch_lightning as pl
+from pytorch_lightning.overrides.fairscale import _FAIRSCALE_AVAILABLE
```

Comment:
Did you make the change to get the full import path?

Reply:
If yes, it means we need to run the meta-package on each commit to verify the imports won't break for a release.

Reply:
Yes, it runs as part of the CI